docs/references/react/use-sign-up.mdx

```diff
@@ -53,33 +53,46 @@ The `useSignUp()` hook gives you access to the [`SignUp`](/docs/references/javas
 
 ## How to use the `useSignUp()` hook
 
-### Check the current state of a sign-up with `useSignUp()`
+### Check the current state of a sign-up
 
-Use the `useSignUp()` hook to access the `SignUp` object and check the current state of a sign-up.
+Use `useSignUp()` to access the `SignUp` object and check the current state of a sign-up.
 
-```tsx {{ filename: 'sign-up-step.tsx' }}
+```tsx {{ filename: 'src/components/SignUpPage.tsx' }}
 import { useSignUp } from '@clerk/clerk-react'
 
 export default function SignUpStep() {
   const { isLoaded, signUp } = useSignUp()
 
   if (!isLoaded) {
-    // Add logic to handle loading state
+    // Handle loading state
     return null
   }
 
-  return <div>The current sign-up attempt status is {signUp.status}.</div>
+  return <div>The current sign-up attempt status is {signUp?.status}.</div>
 }
 ```
 
+### `SignUp` status values
+
 The `status` property of the `SignUp` object can be one of the following values:
 
-| Values | Description |
-| - | - |
-| `complete` | The user has been created and custom flow can proceed to `setActive()` to create session. |
-| `abandoned` | The sign-up attempt will be abandoned if it was started more than 24 hours previously. |
-| `missing_requirements` | A requirement from the [**Email, phone, username**](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username) settings is missing. For example, in the Clerk Dashboard, the password setting is required but a password was not provided in the custom flow. |
+<Properties>
+  - `complete`
+  - `string`
+
+  The user has been created and the custom flow can proceed to `setActive()` to create a session.
+
+  ---
+
+  - `abandoned`
+  - `string`
+
+  The sign-up attempt will be abandoned if it was started more than 24 hours previously.
 
-### Create a custom sign-up flow with `useSignUp()`
```

Comment:
these sections need to stay - they were added a result of user feedback.
we can rename to "Example" if that makes more sense